### PR TITLE
Bug 1378703: No need to show error about editing again when bc is deleted

### PR DESCRIPTION
**Pull request by jhadvig**
Wasn't sure if we shall do it this way(deleting the opposing alert) or just nuke all alerts... but would rather go with this one since its safer.

@spadgett PTAL

### CODE REVIEW

**Comment by spadgett** [comment]
I mean the same alert ID for updated and deleted.

**Comment by jhadvig** [comment]
@spadgett updated

File: app/scripts/controllers/edit/buildConfig.js [hunk 1]
```diff
@@ -198,13 +198,13 @@ angular.module('openshiftConsole')
             // If we found the item successfully, watch for changes on it
             watches.push(DataService.watchObject("buildconfigs", $routeParams.buildconfig, context, function(buildConfig, action) {
               if (action === 'MODIFIED') {
-                $scope.alerts["background_update"] = {
+                $scope.alerts["updated/created"] = {
```
Comment (spadgett) [inline app/scripts/controllers/edit/buildConfig.js]: `"updated/deleted"`?

**Comment by jhadvig** [comment]
@spadgett sorry for that... had a long day
PTAL

**Comment by spadgett** [comment]
[merge]

**Pull request merged**